x-pack/test/functional/apps/ml/anomaly_detection/lens_to_ml.ts

```diff
@@ -0,0 +1,222 @@
+    jobId: string,
+    splitField?: string,
+    aggAndFieldIdentifier?: string
+  ) {
+    await headerPage.waitUntilLoadingHasFinished();
+
+    if (splitField !== undefined) {
+      await ml.jobWizardMultiMetric.assertDetectorSplitExists(splitField);
+      await ml.jobWizardCommon.assertInfluencerSelection([splitField]);
+    }
+
+    if (aggAndFieldIdentifier !== undefined) {
+      await ml.jobWizardCommon.assertAggAndFieldInputExists();
+      await ml.jobWizardCommon.selectAggAndField(aggAndFieldIdentifier, true);
+      await ml.jobWizardCommon.assertAnomalyChartExists('LINE');
+    }
+
+    await ml.testExecution.logTestStep('job creation displays the job details step');
+    await ml.jobWizardCommon.advanceToJobDetailsSection();
+
+    await ml.testExecution.logTestStep('job creation inputs the job id');
+    await ml.jobWizardCommon.assertJobIdInputExists();
+    await ml.jobWizardCommon.setJobId(jobId);
+
+    await ml.testExecution.logTestStep('job creation displays the validation step');
+    await ml.jobWizardCommon.advanceToValidationSection();
+
+    await ml.testExecution.logTestStep('job creation displays the summary step');
+    await ml.jobWizardCommon.advanceToSummarySection();
+
+    await ml.testExecution.logTestStep('job creation creates the job and finishes processing');
+    await ml.jobWizardCommon.assertCreateJobButtonExists();
+    await ml.jobWizardCommon.createJobAndWaitForCompletion();
+
+    await ml.testExecution.logTestStep('job creation displays the created job in the job list');
+    await ml.navigation.navigateToMl();
+    await ml.navigation.navigateToJobManagement();
+
+    await ml.jobTable.filterWithSearchString(jobId, 1);
+
+    await ml.jobTable.assertJobRowJobId(jobId);
+  }
+
+  describe('anomaly charts in dashboard', function () {
+    this.tags(['ml']);
+
+    before(async () => {
+      await ml.testResources.setKibanaTimeZoneToUTC();
+      await ml.securityUI.loginAsMlPowerUser();
+
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/farequote');
+      await kibanaServer.importExport.load(
+        'x-pack/test/functional/apps/ml/anomaly_detection/fixtures/lens_to_ml.json'
+      );
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+      await kibanaServer.importExport.unload(
+        'x-pack/test/functional/apps/ml/anomaly_detection/fixtures/lens_to_ml.json'
+      );
+    });
+
+    beforeEach(async () => {
+      await PageObjects.common.navigateToApp('dashboard');
+    });
+
+    describe('tests create job from lens', () => {
+      let tabsCount = 1;
+
+      afterEach(async () => {
+        if (tabsCount > 1) {
+          await browser.closeCurrentWindow();
+          await retrySwitchTab(0, 10);
+          tabsCount--;
+        }
+      });
+
+      it('can create multi metric job from vis with single layer', async () => {
+        const selectedPanelTitle = 'panel1';
+        const jobId = 'testest4';
```

Comment:
Just a nit, but could we have job ids that give us a bit more context? I'm thinking about error messages in the log, left over jobs after clean-up, etc. - in all of these cases it'd be easier to debug if the job ids were something like e.g. `job_from_lens_COUNTER`.

Reply:
ah yes, I meant to change these IDs before committing

Reply:
https://github.com/elastic/kibana/pull/133899/commits/b05cf28b33b439400f16dcc749d72e7edac877f0